src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IConfigurationRefresher.cs

```diff
@@ -30,8 +30,10 @@ public interface IConfigurationRefresher
 
         /// <summary>
         /// Sets the cached value for key-values registered for refresh as dirty.
-        /// The next call to <see cref="RefreshAsync"/> or <see cref="TryRefreshAsync"/> would cause cached values to be revalidated.
+        /// A random delay is added before the cached value is marked as dirty to reduce potential throttling in case multiple instances refresh at the same time.
+        /// After the cache is marked as dirty, the subsequent call to <see cref="RefreshAsync"/> or <see cref="TryRefreshAsync"/> would cause cached values to be revalidated.
```

Comment:
I didn't really understand the last line. I thought the first two lines gave a good summary though. Do we need the last line?

Reply:
I will remove the last line. The only additional information it adds is that the refresh method needs to be invoked for the revalidation to happen, and that could be considered to be implicit from the method name and the interface context.